robocop/checkers/naming.py

```diff
@@ -370,6 +375,10 @@ def visit_VariableSection(self, node):  # noqa
                             col=token.col_offset)
 
     def visit_KeywordCall(self, node):  # noqa
+        for token in node.get_tokens(Token.ASSIGN):
```

Comment:
What get_tokens will return if there are not assign tokens? I think it will be None and it will break code

Reply:
It will be an empty list and it will just not iterate the for loop.